File: jvm2/src/Model/Abstract/BlankTile.kt
package Model.Abstract

import Model.Location
import Model.Move

class BlankTile(location: Location) : Tile() {
    override var loc = location
    override var occ = false
    override var use = false
    override var drive = false
    override var traverse = true
    override var owner = 0
    override var blank = true
    override var icon = "?"
    override var minCost = 0
    override var gasCost = 0
    override var nrgCost = 0
    override var wait = 0
    override var health = 0
    override var minSal = 0
    override var gasSal = 0
    override var nrgSal = 0
}

File: dotnet/src/Model/Abstract/BlankTile.cs
using System;

namespace PlanetCS1 {
    class BlankTile : Tile {
        public BlankTile(Location location) {
            this.loc = location;
            this.occ = false;
            this.use = false;
            this.drive = false;
            this.traverse = true;
            this.owner = 0;
            this.blank = true;
            this.icon = "?";
            this.minCost = 0;
            this.gasCost = 0;
            this.nrgCost = 0;
            this.wait = 0;
            this.health = 0;
            this.minSal = 0;
            this.nrgSal = 0;
        }
    }
}